packages/fxa-settings/src/styles/custom/animations.scss

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@keyframes rotate {
+  0% {
+    transform: rotate(0);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.animate-timing-linear {
+  animation-timing-function: linear;
+}
+
+.animate-duration-800 {
+  animation-duration: 800ms;
+}
+
+.animate-repeat-infinite {
+  animation-iteration-count: infinite;
+}
+
+.animate-rotate {
+  animation-name: rotate;
+}
```

Comment:
In theory these are meant to be globally available, and not custom/specific to a given component -- should they exist in `custom/`? What is `styles/` if not a directory of custom CSS files? 😉

Reply:
I could take them out of this dir if you think that's preferable, we might end up with a decent amount of files under `styles/` though. 🤔

Reply:
Is that bad?

Reply:
I think that's OK until we have a lot of files. Maybe we won't get there, I'll pull them out of `custom/`.